<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>611</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="crossvalidateButton">
       <property name="text">
        <string>Cross Validate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="trainButton">
       <property name="text">
        <string>Train</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="predictButton">
       <property name="text">
        <string>Predict</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>clear (output)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>401</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="outputTextEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>input</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="inputTextEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>classifier</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="classifierListWidget">
       <item>
        <property name="text">
         <string>LogisticRegression</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LinearSVC</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>132</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>dataset  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="datasetListWidget">
       <item>
        <property name="text">
         <string>aggression</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>attack</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>toxicity</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>analyzer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="analyzerListWidget">
       <item>
        <property name="text">
         <string>char</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>char_wb</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>word</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>max ngram             </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="maxGramLineEdit">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>min ngram              </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="minGramLineEdit">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>CV folds                 </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cv_foldsLineEdit">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>max features          </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="maxFeatLineEdit">
       <property name="text">
        <string>1000</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>label_2</zorder>
   <zorder>horizontalLayoutWidget_8</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>horizontalLayoutWidget_6</zorder>
   <zorder>horizontalLayoutWidget_5</zorder>
   <zorder>horizontalLayoutWidget_4</zorder>
   <zorder>label_2</zorder>
   <zorder>maxFeatLineEdit</zorder>
   <zorder>horizontalLayoutWidget_3</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>horizontalLayoutWidget_7</zorder>
   <zorder>horizontalLayoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>637</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWikipedia_Talk_Data">
    <property name="title">
     <string>Wikipedia Talk Data</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuWikipedia_Talk_Data"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
